README.md

```diff
@@ -202,6 +203,7 @@ see this [link](https://github.com/census-instrumentation/opencensus-java/tree/m
 [maven-url]: https://maven-badges.herokuapp.com/maven-central/io.opencensus/opencensus-api
 [gitter-image]: https://badges.gitter.im/census-instrumentation/lobby.svg
 [gitter-url]: https://gitter.im/census-instrumentation/lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+[cloud-versioning]: https://github.com/GoogleCloudPlatform/google-cloud-java#versioning
```

Comment:
I think these definitions are specific to google-cloud-java.  Is there a more standard definition we could use?  If not, we could just describe the level of stability that we are aiming for in a few sentences.

Reply:
Didn't find an official definition for the versioning. I saw that opencensus-php-exporter-stackdriver copied the definition of versioning in their [README](https://github.com/census-ecosystem/opencensus-php-exporter-stackdriver#versioning), I'll copy that to our README too.